docs/src/main/sphinx/security/tls.rst

```diff
@@ -259,15 +259,37 @@ re-enable it by setting:
 
   http-server.authentication.allow-insecure-over-http=true
 
-Test configuration
-^^^^^^^^^^^^^^^^^^
+.. _verify_tls:
 
-To test your configuration settings, restart the server and try to connect to it
-with the Trino :doc:`CLI </client/cli>` or :doc:`Web UI
-</admin/web-interface>`, using a URL that begins with ``https://``.
+Verify configuration
+^^^^^^^^^^^^^^^^^^^^
 
-Now that TLS is configured, go back and :doc:`configure the authentication
-</security>` method for your server.
+To verify TLS/HTTPS configuration, log in to the :doc:`Web UI
```

Comment:
may add a "you should" in here otherwise it reads as the instructions, which are below, and not a summary